docs/permissions.mdx

```diff
@@ -0,0 +1,205 @@
+
+### odigos-data-collection
+
+| APIGroups | Resources | Resource Names | Verbs |
+|---|---|---|---|
+| \* | nodes/stats<br />nodes/proxy | \* | get<br />list |
+| \* | pods<br />namespaces | \* | get<br />list<br />watch |
+| apps | replicasets<br />deployments<br />daemonsets<br />statefulsets | \* | get<br />list<br />watch |
+| \* | endpoints | \* | get<br />list<br />watch |
+
+### odigos-instrumentor
+
+| APIGroups | Resources | Resource Names | Verbs |
+|---|---|---|---|
+| \* | nodes | \* | list<br />watch<br />get |
+| \* | namespaces | \* | list<br />watch<br />get |
+| \* | pods | \* | list<br />watch<br />get |
+| apps | daemonsets | \* | get<br />list<br />watch<br />update<br />patch |
+| apps | deployments | \* | get<br />list<br />watch<br />update<br />patch |
+| apps | statefulsets | \* | get<br />list<br />watch<br />update<br />patch |
+| apps | statefulsets/finalizers<br />deployments/finalizers<br />daemonsets/finalizers | \* | update |
+| operator.odigos.io | odigos/finalizers | \* | update |
+| odigos.io | instrumentedapplications | \* | delete<br />get<br />list<br />watch |
+| odigos.io | instrumentationconfigs/status | \* | get<br />patch<br />update |
+| odigos.io | instrumentationconfigs | \* | create<br />delete<br />get<br />list<br />patch<br />update<br />watch |
+| odigos.io | sources | \* | create<br />delete<br />get<br />list<br />patch<br />update<br />watch |
+| odigos.io | sources/finalizers | \* | update |
+
+### odiglet
+
+| APIGroups | Resources | Resource Names | Verbs |
+|---|---|---|---|
+| \* | pods | \* | get<br />list<br />watch |
+| \* | pods/status | \* | get |
+| \* | pods/finalizers | \* | update |
+| \* | nodes | \* | get<br />list<br />watch<br />patch |
+| odigos.io | instrumentationinstances | \* | create<br />get<br />list<br />patch<br />update<br />watch<br />delete |
+| odigos.io | instrumentationinstances/status | \* | get<br />patch<br />update |
+| odigos.io | instrumentationconfigs | \* | get<br />list<br />watch<br />patch<br />update |
+| odigos.io | instrumentationconfigs/status | \* | get<br />patch<br />update |
+
+### odigos-scheduler
+
+| APIGroups | Resources | Resource Names | Verbs |
+|---|---|---|---|
+| odigos.io | instrumentationconfigs | \* | get<br />list<br />watch |
+
+### odigos-ui
+
+| APIGroups | Resources | Resource Names | Verbs |
+|---|---|---|---|
+| \* | namespaces | \* | get<br />list<br />patch |
+| apps | deployments<br />statefulsets<br />daemonsets | \* | get<br />list |
+| apps | replicasets | \* | get<br />list |
+| \* | services | \* | get<br />list |
+| \* | pods | \* | get<br />list<br />watch |
+| odigos.io | instrumentationconfigs<br />instrumentationinstances | \* | get<br />list<br />watch |
+| odigos.io | sources | \* | get<br />list<br />patch<br />create<br />delete |
+
+## Roles
+
+Below are the Roles used by Odigos components.
```

Comment:
Suggestion: mention that these roles apply to the namespace where odigos iss installed (`odigos-system` or other value set during install).

I guess it's obvious, but we can make it explicit and avoid any confusions.